recipes/libdc1394/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+
+class Libdc1394Conan(ConanFile):
+    name = "libdc1394"
+    license = "LGPL-2.1"
```

Comment:
```suggestion
    license = "LGPL-2.1-or-later"
```

LGPL-2.1 is deprecated. dc1394 license is "-or-later".

https://spdx.org/licenses/LGPL-2.1-or-later.html